httpmw/operationid/operationid.go

```diff
@@ -0,0 +1,92 @@
+package operationid
+
+import (
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"google.golang.org/grpc/metadata"
+)
+
+type contextKey string
+
+const (
+	CorrelationIDKey        contextKey = "correlationID"
+	ARMClientRequestIDKey   contextKey = "armClientRequestID"
+	GraphClientRequestIDKey contextKey = "graphClientRequestID"
+	ClientSessionIDKey      contextKey = "clientSessionID"
+	ClientApplicationIDKey  contextKey = "clientApplicationID"
+	ClientPrincipalNameKey  contextKey = "clientPrincipalName"
```

Comment:
Suggest to use the same key as what is in the http header. Why introduce unnecessary aliases?

E.g., name it the same as `x-ms-correlation-request-id`

Reply:
followed the same format as the toolkit logger in aks-rp, I removed the aliases since they are not necessary in our case

Reply:
What do you mean not nicely formatted? It is just key value pairs. What format do we expect?

You may chat with Christine to see if there are any naming restrictions in Kusto (- is OK. https://learn.microsoft.com/en-us/kusto/query/schema-entities/entity-names?view=microsoft-fabric) or Geneva. If so, document them here and create alias. Otherwise, extra alias is a pain for human to map them.

Reply:
It is to keep the log key the same as before. If users want to change it, they can use custom Extractor function.

Reply:
as discussed keeping the formatting consistent for aks engineers, user can customize keys if they wish to do so